docs/js-core-1/week-1/lesson.md

```diff
@@ -239,6 +233,106 @@ Using online documentation, what other things can you do with the `Math` library
 
 Pick one thing on your table that you can do other than `Math.round` and prepare an explanation for the rest of the class.
 
+
+You can also use expressions with string interpolation or as a `return` value.
+
+```js
+console.log(`2 + 4 is ${2 + 4}`); // 2 + 4 inside the string template evaluates to 6
+
+function double(num) {
+  return num * 2; // expression num * 2 being returned from this function
+}
+```
+
+## Statements and expressions
+
+In JavaScript, there are **expressions** and **statements**. 
+We will use these words frequently to describe code.
+### Statement
+
+A statement is some code that typically carries out an instruction. You have encountered statements already:
+
+```js
+const greeting = "Hello Code Your Future!";
+```
+
+The above piece of code is a **variable declaration**, which is also a statement. 
+This statement is just an instruction to store the value `"Hello Code Your Future!"` in the variable `greeting`.
+
+Statements end in a semi-colon, though they can be omitted in JavaScript.
```

Comment:
We did discuss that understanding of what "evaluating an expression" means is a useful tool for instructors, and I do think we refer to things like "if statements" fairly often. So having a definition is perhaps a good idea.

I would lean towards moving the statements section below the expressions section. I think it may be easier to define statements in context of expressions: "If expressions are single units of JavaScript that the engine can evaluate, statements can contain one or more different expressions, and are executed by the engine to perform an operation" ([source](https://flaviocopes.com/javascript-statements/))

Reply:
Also, what do you think about leaving out the point about semi-colons being optional?

Benefits:

- Prettier enforces semis by default, so trainees shouldn't have to worry about them
- JS' automatic semi-colon insertion is infamously weird, and discussing it will be distracting

Drawbacks:

- Some volunteers (like myself) are in the habit of leaving off semis, so it tends to come up when live-coding

I am a bit on the fence, but lean towards leaving it out.